software/config/val/robot.cfg

```diff
@@ -1,6 +1,6 @@
 INCLUDE = [
   "core_config.cfg",
-  "multisense.cfg",
+  "multisense_D_50.cfg",
```

Comment:
Opened #100 to track this to make this PR non-blocking. The ${VAL_UNIT} variable is probably a good approach for this, need to be incorporated in director then as well. Otherwise PR is ready to go